.travis.yml

```diff
@@ -3,10 +3,12 @@ language: ruby
 before_script: .travis/before.sh
 install: bundle install --deployment --without development production
 rvm: 2.1.3
-cache: bundle
+cache: bundler
```

Comment:
Since we override the install step and don't provide a `--path` I don't think this would even work?

Perhaps we should switch install step to `bundler_args: --deployment --without development production`